domain/builders/atrium-domain-builders-jvm/src/main/kotlin/ch/tutteli/atrium/domain/builders/creating/ChronoLocalDateAssertionsBuilder.kt

```diff
@@ -0,0 +1,24 @@
+@file:Suppress("OVERRIDE_BY_INLINE", "NOTHING_TO_INLINE")
+
+package ch.tutteli.atrium.domain.builders.creating
+
+import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.core.polyfills.loadSingleService
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.creating.ChronoLocalDateAssertions
+import ch.tutteli.atrium.domain.creating.chronoLocalDateAssertions
+import java.time.chrono.ChronoLocalDate
+import java.time.chrono.ChronoLocalDateTime
+
+/**
+ * Delegates inter alia to the implementation of [ChronoLocalDateAssertionsBuilder].
+ * In detail, it implements [ChronoLocalDateAssertions] by delegating to [chronoLocalDateAssertions]
+ * which in turn delegates to the implementation via [loadSingleService].
+ */
+object ChronoLocalDateAssertionsBuilder : ChronoLocalDateAssertions {
+    override fun <T : ChronoLocalDateTime<ChronoLocalDate>> isBefore(
```

Comment:
- add `inline` in addition
- move everything on one line
- remove the return type (as we delegate to an interface we don't have to specify the return type again, no chance to leak something)
- you probably guessed it, ChronoLocalDate :)